### PR TITLE
T10342905

### PR DESCRIPTION
**Pull request by catalina-macovei**
Merged development into main and added submissions.

### CODE REVIEW

File: Controllers/ExercisesController.cs [hunk 1]
```diff
@@ -461,15 +467,65 @@ public async Task<IActionResult> ExecuteCode(int id, string solution, string lan
...
+                if (response.IsSuccessStatusCode)
+                {
+                    var jsonResponse = await response.Content.ReadAsStringAsync();
+                    var result = JsonConvert.DeserializeObject<Dictionary<string, object>>(jsonResponse);
+
+                    int passedTests = 0;
+                    int failedTests = 0;
+
+                    /*
+                    foreach (var test in result)
```
Comment (Alexandra06T) [inline Controllers/ExercisesController.cs:483]: Score field is always 0, no matter the results of the tests. Probably you should fix this function in order to calculate the score correctly.

File: Views/Solutions/Show.cshtml [hunk 2]
```diff
@@ -1,31 +1,41 @@
 ﻿@model Developer_Toolbox.Models.Solution
-<section class="p-3">
-    <h1 class="">Solution:</h1>
-    <br />
 
-    <div class="card w-50">
```
Comment (Alexandra06T) [inline Views/Solutions/Show.cshtml:6]: Displaying the solutions in cards takes a lot of spaces and becomes difficult to understand for the user. You should probably list them, including only some relevant information.

**Review by Alexandra06T — CHANGES_REQUESTED** [review]

File: Controllers/ExercisesController.cs (hunk 1 above)
Comment (catalina-macovei) [reply Controllers/ExercisesController.cs]: I've fixed the issue, thank you!

File: Views/Solutions/Show.cshtml (hunk 2 above)
Comment (catalina-macovei) [reply Views/Solutions/Show.cshtml:6]: I'll try now to fix this

**Review by Alexandra06T — APPROVED** [review]
Good job! The code is now ready to be merged.

**Pull request merged**